ddtrace/_logger.py

```diff
@@ -32,14 +37,18 @@ def configure_ddtrace_logger():
 
     """
     ddtrace_logger = logging.getLogger("ddtrace")
-
     _configure_ddtrace_debug_logger(ddtrace_logger)
     _configure_ddtrace_file_logger(ddtrace_logger)
 
 
 def _configure_ddtrace_debug_logger(logger):
     debug_enabled = asbool(os.environ.get("DD_TRACE_DEBUG", "false"))
-    if debug_enabled:
+    logs_injection_enabled = asbool(os.environ.get("DD_LOGS_INJECTION", "false"))
+
+    # Logs injection enabled requires a handler present on ddtrace
+    if debug_enabled or logs_injection_enabled:
+        if not os.environ.get("DD_TRACE_LOG_FILE"):
+            logger.addHandler(logging.StreamHandler())
```

Comment:
should we have a function similar to `_configure_ddtrace_file_logger`, but which configures the stream handler?

should we always configure the stream handler? (with a default log level of like `WARNING` or something?)

Reply:
Log levels for unconfingured loggers are `WARNING` by default. To your point, I've changed this logic to always create a `StreamHandler` on startup (but unconfigured) so that it's available to be configured for trace debugging and log injections down the line.